### DISCUSSION TITLE
How can i use sql views ?

### DISCUSSION THREAD

**Discussion by EtienneProthon** [SeaQL/sea-orm#814] [Q&A] [answered]
I am wondering if there is a way to create and use sql view with sea-orm.

**Comment by ikrivosheev** [comment]
@EtienneProthon hello! I am working on this: https://github.com/SeaQL/sea-query/issues/179

**Reply by EtienneProthon** [reply]
Is there anything i can do to help you on this PR ? What is the current state ?

**Comment by billy1624** [comment, marked as answer]
Hey @EtienneProthon, SeaORM rely on SeaQuery to construct all necessary SQL statement. Since the support of view statements are under development. You cannot create / modify view with SeaORM nor SeaQuery.

But I think you can still query view with SeaORM just like what you can do to a ordinary table.

**Comment by tyt2y3** [comment]
Yes you can select from view, but not creating them for now

**Comment by EtienneProthon** [comment]
Yes i know i can select from them, for the moment i will create them from raw on the migration part. Thanks for your quick answers !

**Reply by izolyomi** [reply]
I've just found this discussion facing the same issue.
At the moment I had to start with an existing database and I'm only generating the entities without migration, but I can't see any support for views in `sea-orm-cli`. I'm still getting familiar with sea-orm, does someone maybe has a link to an example showing how views still can be queried?

**Reply by EtienneProthon** [reply]
Just do like you would with normal table (create the model etc...) it will generate the sql request to query it but don't try to update or delete because as it's a view it will not work.

**Reply by billy1624** [reply]
Thanks!! @EtienneProthon
